src/style.css

```diff
@@ -1 +1,81 @@
 /* add styles here */
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300px;
+  line-height: 21.86px;
+  text-align: left;
+}
+
+@font-face {
```

Comment:
The `@font-face` rule should define the font-family name once and specify different `src` and `font-weight` or `font-style` for different font files. Combine the `src` urls into one rule and differentiate with `font-weight` and `font-style` as needed.